plugin/ocgrpc/server.go

```diff
@@ -36,12 +31,20 @@ type ServerHandler struct {
 	// NoStats may be set to disable recording OpenCensus Stats around each
 	// gRPC method.
 	NoStats bool
+
+	// Set to true if trace metadata should be read from inbound requests.
+	// Only set this if the current service will only be invoked by
+	// services under your control. Otherwise, a malicious user might be able
+	// to trigger traces in your backend.
+	//
+	// If not set, any trace information in the inbound request will be ignored.
+	TrustCallerMetadata bool
```

Comment:
Why would anyone use the stats handler directly in a public API without taking any security considerations? The security concern here is beyond what OpenCensus propagates but applies to all metadata.

Are we expecting a public-facing server to disable this?

Why not default being the opposite?

s/TrustCallerMetadata/TrustCaller?

// TrustCallerMetadata trust the trace metadata of the incoming calls.
// ... 
TrustCallerMetadata bool

golint, godoc should start with exported identifier.

Reply:
changed to default trust for gRPC.
yes, public-facing servers should not trust tracing metadata I think. Ultimately it's up to the developer to decide but this gives them a way to not trust but still create server-side spans.